packages/config/src/configs/moonbeam.ts

```diff
@@ -724,6 +725,21 @@ export const moonbeamConfig = new ChainConfig({
         balance: BalanceBuilder().substrate().system().account(),
       },
     }),
+    new AssetConfig({
+      asset: bncs,
+      balance: BalanceBuilder().evm().erc20(),
```

Comment:
```suggestion
      balance: BalanceBuilder().substrate().assets().account(),
```
BNCS is not a ERC20, so the function to get the balance is a substrate call, not a smart contract call